creator-node/src/routes/files.js

```diff
@@ -120,8 +124,16 @@ const streamFromFileSystem = async (
       )
     }
 
-    // Set the CID cache-control so that client caches the response for 30 days
-    res.setHeader('cache-control', 'public, max-age=2592000, immutable')
+    // If content is NOT premium, set the CID cache-control so that client caches the response for 30 days.
+    // Otherwise, set cache-control to no-cache.
+    if (req.premiumContent && req.premiumContent.isPremium) {
+      req.logger.info(
+        'Setting the no-cache cache-control response header for premium content.'
+      )
+      res.setHeader('cache-control', 'no-cache')
+    } else {
+      res.setHeader('cache-control', 'public, max-age=2592000, immutable')
```

Comment:
hmm what's this immutable. gotta make sure this 'no-cache' setting works properly for premium tracks

Reply:
`immutable` should just imply that the content wont ever change, thus no need for a refresh on it. I think it should be fine here

Reply:
with the max age I think responses with this header with eventually expire